app/components/smart-banner-test.js

```diff
@@ -0,0 +1,190 @@
+import Ember from 'ember';
+import Thumbnailer from '../modules/thumbnailer';
+import {track, trackActions} from '../utils/track';
+import {system, standalone} from '../utils/browser';
+
+const {
+	$,
+	Component,
+	computed,
+	get,
+	inject,
+	String: {htmlSafe},
+	run,
+} = Ember;
+
+/**
+ * Component for a custom Smart Banner
+ * it's visible only for Android devices
+ * iOS has its own native smart banner - no need to render it there
+ */
+export default Component.extend({
+	classNames: ['smart-banner-test'],
+	classNameBindings: ['noIcon'],
+
+	wikiVariables: inject.service(),
+	currentUser: inject.service(),
+
+	options: {
+		// Language code for App Store
+		appStoreLanguage: 'us',
+
+		// Duration to hide the banner after close button is clicked (0 = always show banner)
+		daysHiddenAfterClose: 15,
+
+		// Duration to hide the banner after it is clicked (0 = always show banner)
+		daysHiddenAfterView: 30,
+	},
+	day: 86400000,
```

Comment:
consider renaming to `dayInSeconds` or sth similar

Reply:
👍